pysatNASA/instruments/omni_hro.py

```diff
@@ -201,13 +193,38 @@ def load(fnames, tag=None, inst_id=None, file_cadence=pds.DateOffset(months=1),
     return data, meta
 
 
-# ----------------------------------------------------------------------------
-# Local functions
+# Local Functions (deprecated)
+
+def deprecated(func):
+    """Warn users that function has moved locations.
+
+    Decorator function for deprecation warnings.
```

Comment:
🎵 Soon may a decorator come 🎵
🎶 To deprecate functions that shouldn't be run 🎶
🎵 And when the testing is done 🎵
🎶 We'll take our leave and go 🎶